kubetest/main.go

```diff
@@ -515,15 +515,26 @@ func prepareGcp(o *options) error {
 	if err := migrateGcpEnvAndOptions(o); err != nil {
 		return err
 	}
-	if o.provider == "gke" && o.gcpNodeImage != "" {
+	if o.provider == "gke" {
+		if o.gcpNodeImage == "" {
+			o.gcpNodeImage = "container_vm"
+		}
+
 		// TODO(kubernetes/test-infra#3307): Should disappear when
 		// "bash" deployment is gone.
 		if o.deployment == "bash" {
 			os.Setenv("KUBE_GKE_IMAGE_TYPE", o.gcpNodeImage)
 		}
+
 		// TODO(kubernetes/test-infra#3536): This is used by the
 		// ginkgo-e2e.sh wrapper.
-		os.Setenv("NODE_OS_DISTRIBUTION", o.gcpNodeImage)
+		nod := o.gcpNodeImage
+		if nod == "container_vm" {
+			// gcloud container clusters create understands
+			// "container_vm", e2es understand "debian".
+			nod = "debian"
```

Comment:
`container_vm` is definitely getting deprecated soon, so I'm not *that* worried about these jobs.

This gets plumbed clear to https://github.com/kubernetes/kubernetes/blob/02b520f0a40be2056d91fc0661c2b4fdb2694c30/test/e2e/framework/test_context.go#L217 by https://github.com/kubernetes/kubernetes/blob/02b520f0a40be2056d91fc0661c2b4fdb2694c30/hack/ginkgo-e2e.sh#L154